app/ui/browser/actions/external.js

```diff
@@ -227,7 +228,7 @@ export function close() {
  * Perform the search if the user just pressed return
  */
 export function inPageSearch(ev) {
-  if (ev.keyCode === 13) {
+  if (ev.keyCode === KEY.ENTER) {
```

Comment:
What if we did `ev.key === "Enter"`, `ev.key === "KeyF"`, `ev.key === "Escape"`, etc instead of making constants?